modules/tile-converter/test/i3s-converter/helpers/geometry-converter.spec.ts

```diff
@@ -592,6 +592,84 @@ test('tile-converter(i3s)#convertB3dmToI3sGeometry - array of UINTxx should be c
   t.end();
 });
 
+test('tile-converter(i3s)#convertB3dmToI3sGeometry - should convert 64-bit attributes to strings', async (t) => {
```

Comment:
Looks like this test is from another PR...

Reply:
No idea why this test is shown as a new one. It has been recently added to master. It looks like this review was done before the merge of that PR. Now it looks good.